forge/postoffice/templates/VerifyEmail.js

```diff
@@ -3,13 +3,16 @@ module.exports = {
     text:
 `Hello, {{{safeName.text}}},
 
-Use the link below to verify your email address.
+Please use this code to verify your email address:
 
-{{{ confirmEmailLink }}}
+{{{ token.token }}}
+
+Do not share this code with anyone else.
```

Comment:
```suggestion
Do not share this code with anyone else.

This token will expire in 30 minutes.
```


Suggestion only: Inform user of expiry duration?